src/lib/stores/sdk.ts

```diff
@@ -13,7 +13,8 @@ import {
     Users
 } from '@aw-labs/appwrite-console';
 
-const endpoint = VARS.APPWRITE_ENDPOINT ?? `${globalThis?.location?.origin}/v1`;
+export const endpoint = VARS.APPWRITE_ENDPOINT ?? `${globalThis?.location?.origin}/v1`;
```

Comment:
I think it might make more sense to use `sdkForConsole.client.config.endpoint` for this instead of exporting 👍🏻

Reply:
Done!